tests/client-dbus/README.rst

```diff
@@ -27,3 +27,8 @@ Contributing
 ------------
 Issues suggesting tests or pull requests that extend the existing test suite
 are welcome.
+
+The code style standard is PEP8.  Travis CI runs a compliance test on
+all pull requests via yapf.  Please auto format code before opening a pull 
+request via "make fmt".  For any cases that hand formatting is more clear, 
```

Comment:
Could you leave out the last sentence? It's usually just easier to obey the autoformatting than to try to override it.

Reply:
Ok